gateway/src/service.rs

```diff
@@ -223,9 +225,15 @@ impl GatewayService {
 
         debug!(target_url, "routing control");
 
+        let headers = req.headers_mut();
+        headers.append(
+            "X-Forwarded-Account-Name",
```

Comment:
In #484 I have added `X-Shuttle-Project`. We should standardise these now. Which is better:
- `X-Forwarded-Project` or
- `X-Shuttle-Account-Name`?

Reply:
I have no opinion, just went with `X-Forwarded` since it seems well-known for proxies. But these are also very shuttle specific so the `X-Shuttle` makes sense :thinking: 

These will never appear in any web traffic so let's go with the `X-Shuttle` and keep these internal